pkg/datapath/linux/routing/info.go

```diff
@@ -77,9 +86,16 @@ func parse(gateway string, cidrs []string, macAddr string, masquerade bool) (*Ro
 		return nil, fmt.Errorf("invalid mac: %s", macAddr)
 	}
 
+	parsedIfaceNum, err := strconv.ParseInt(ifaceNum, 10, 64)
```

Comment:
I'd suggest to do:

```suggestion
	parsedIfaceNum, err := strconv.Atoi(ifaceNum)
```

To parse it as `int` from the beginning, instead of parsing as `int64` and then converting to `int`. I think that should fix the warning below.